src/context/alert.tsx

```diff
@@ -15,28 +15,28 @@ export enum WARNING_MESSAGE {
 }
 
 export enum ERROR_MESSAGE {
-  DUPLICATE_IDENTIFIER_INPUT = "Enter unique code details.",
+  DUPLICATE_IDENTIFIER_INPUT = "Enter or scan a different code.",
   DUPLICATE_POD_INPUT = "Scan another item that is not tagged to any ID number.",
-  INVALID_IDENTIFIER_INPUT = "Enter or scan valid code details.",
-  MISSING_IDENTIFIER_INPUT = "Enter or scan code details.",
-  INVALID_VOUCHER_INPUT = "Enter valid voucher code details.",
-  MISSING_VOUCHER_INPUT = "Enter voucher code details.",
-  INVALID_POD_INPUT = "Scan valid device code.",
-  MISSING_POD_INPUT = "Scan device code.",
-  INVALID_PHONE_NUMBER = "Enter valid contact number.",
-  INVALID_COUNTRY_CODE = "Enter valid country code.",
-  INVALID_PHONE_AND_COUNTRY_CODE = "Enter valid country code and contact number.",
+  INVALID_IDENTIFIER_INPUT = "Enter or scan a valid code.",
+  MISSING_IDENTIFIER_INPUT = "Enter or scan a code.",
+  INVALID_VOUCHER_INPUT = "Enter a valid voucher code.",
+  MISSING_VOUCHER_INPUT = "Enter a voucher code.",
+  INVALID_POD_INPUT = "Scan a valid device code.",
+  MISSING_POD_INPUT = "Scan a device code.",
+  INVALID_PHONE_NUMBER = "Enter a valid contact number.",
+  INVALID_COUNTRY_CODE = "Enter a valid country code.",
+  INVALID_PHONE_AND_COUNTRY_CODE = "Enter a valid country code and contact number.",
```

Comment:
for the changes in the error messages here, would have to make the same changes to the relevant test cases too!